src/overlays/actors/ovl_Bg_Dy_Yoseizo/z_bg_dy_yoseizo.h

```diff
@@ -10,6 +10,8 @@ struct BgDyYoseizo;
 
 typedef void (*BgDyYoseizoActionFunc)(struct BgDyYoseizo*, GlobalContext*);
 
+#define BG_DY_YOSEIZO_EFFECTS_COUNT 200
```

Comment:
Pretty minor but I think `_EFFECT_COUNT` (singular) would be more consistent with the rest of the codebase (with the added benefit of being shorter)